mypy/test/testpep561.py

```diff
@@ -56,11 +56,18 @@ class SimpleMsg(Enum):
     msg_dne = "{tempfile}:3: error: Module 'typedpkg' has no attribute 'dne'"
     msg_list = "{tempfile}:5: note: Revealed type is 'builtins.list[builtins.str]'"
     msg_tuple = "{tempfile}:5: note: Revealed type is 'builtins.tuple[builtins.str]'"
+    msg_any = "{tempfile}:5: note: Revealed type is 'Any'"
 
 
 class NamespaceMsg(Enum):
     cfm_beta = ("{tempfile}:4: error: Cannot find implementation or library stub for module named "
                 "'typedpkg_ns.ns.dne'")
+    missing_typedpkg_module = ("{tempfile}:3: error: Cannot find implementation or library stub"
```

Comment:
Instead of messing with string formatting, I would make your own Enum for these values.

Reply:
Wow glad to be back. I am not too sure what too change, since the messages are currently added to the proper class without formatting (that was a different idea).
I've added:
- missing_typedpkg_module
- missing_typedpkg_sample_module
- help_note2 (copy of help_note)

Anything specific worth improving?

Reply:
@ethanhs I'd gladly hear your thoughts.

Reply:
Hi @davidzwa, I actually restructured the tests to be a bit more data driven. Sorry for making extra work for you. If you rebase on master and add your test to  https://github.com/python/mypy/blob/master/test-data/unit/pep561.test, you should just be able to put the test there and then put the messages in the `[out]` section (please also put `# pkg: typedpkg` at the beginning of your test, so that we can check it is not finding the installed package). Hopefully this is a bit clearer than before.

Reply:
I am currently adding the test. You can see my progress in the check list at the top.

Reply:
@ethanhs in hindsight: how do I add the configuration file now? I remember from earlier that I can add a line `# cmd: mypy --no-site-packages` to trigger the flag, but the `mypy.conf` I cannot create.

I am ok with just testing the no-site-packages command line flag and leaving the Options class untested, if that's ok?